pkg/boot/netboot/netboot.go

```diff
@@ -76,6 +76,12 @@ func getBootImage(schemes curl.Schemes, uri *url.URL, mac net.HardwareAddr, ip n
 		return nil, fmt.Errorf("failed to parse pxelinux config: %v", err)
 	}
 
+	if pc.DefaultEntry == "" {
+		pc.DefaultEntry = "LinuxImage"
```

Comment:
when no label is specified, do both packages default to "LinuxImage"? that seems wrong. 

curEntry is not LinuxImage at this point, ever, by default: https://github.com/u-root/u-root/blob/6bcd1cda59967cb3cd82d00fc963559ffbb814f6/pkg/boot/syslinux/syslinux.go#L203

so how does this work?

Reply:
I don't know. I am wondering: should the pxeboot.go set a DefaultEntry? That seems better?

Reply:
ah, not needed. I figured out a config that our pxeboot will like. I remove this bit.